src/components/AddToCart.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useCart } from '../state/CartProvider';
+
+export default function AddToCart({ product }) {
+  const { addToCart } = useCart();
+
+  const handleClick = () => {
+    console.log('Adding to cart:', product); // Debugging log
+    addToCart(product);
+  };
+
+  return (
+    <button
+      className="f6 link dim br3 ba bw1 ph3 pv2 mb2 dib black"
+      onClick={handleClick}
+    >
+      Add to Cart
+    </button>
+  );
+}
```

Comment:
The [`src/components/Button.jsx`](https://github.com/CallmeArs1/KhanArselanWeek07/blob/main/src/components/Button.jsx) would have sufficed here. Per the instructions, it accepts a `handleClick` prop which would handle the add to cart logic.